### PR TITLE
Re-simplifies corpse examination messages.

### PR DESCRIPTION
**Pull request by itseasytosee**
## About The Pull Request
Dead examination messages are just like they were before. If they are revivable, they are simply dead and lifeless. If they are not, their soul is departed. No needless text.

## Why It's Good For The Game
Fixes stay, confusion goes away.

## Changelog
:cl: itseasytosee
qol: death examination are less confusing
/:cl:

### CODE REVIEW

**Comment by itseasytosee** [comment]
Rohesie told me this was cool.

**Comment by Fikou** [comment]
the check you made is
```
if(x)
  a
if(y)
  b
a
```
it makes no sense

File: code/modules/mob/living/carbon/human/human_helpers.dm [hunk 1]
```diff
@@ -227,11 +227,12 @@
 	var/t_He = p_they(TRUE)
 	var/t_his = p_their()
 	var/t_is = p_are()
+	var/defalt_message = "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life...</span>"
```
Comment (coiax) [inline code/modules/mob/living/carbon/human/human_helpers.dm:230]:
```suggestion
	var/default_message = "<span class='deadsay'>[t_He] [t_is] limp and unresponsive; there are no signs of life...</span>"
```

Will need to change the usages as well.

**Comment by itseasytosee** [comment]
true story

**Review by Mothblocks — APPROVED** [review]

**Review by coiax — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is now above zero (0). Feel free to make Feature/Tweak/Balance PRs.

**Pull request merged**